environment/deployments/science-platform/env/dev-gke.tfvars

```diff
@@ -68,3 +68,13 @@ node_pools = [
     disk_type          = "pd-ssd"
   }
 ]
+
+node_pools_labels = {
+  core-pool = {
+    dask= ""
```

Comment:
I'm curious why you nulling these labels instead of assigning them.  Are they set to "ok" elsewhere?

Reply:
Yes, this is the threaded discussion I've been having in #ops-google-idf.  I am happy with the default having them all set, but in this particular instance we clearly want to force all the dask workflow into one place.

Reply:
Sounds good